### ISSUE TITLE
loss function of f-vae

### ISSUE THREAD

**Issue by chshm** [bojone/flow#9]
f-vae的recon_loss = 0.5 * K.sum(K.mean(x_recon**2, 0)) + 0.5 * np.log(2*np.pi) * np.prod(K.int_shape(x_recon)[1:])的后半部分是怎么来的
